src/app/pages/datasets/components/dataset-management/dataset-management.component.html

```diff
@@ -1,78 +1,86 @@
-<div class="container" fxLayoutGap="16px">
-  <div class="table-container">
-    <ix-fake-progress-bar
-      class="loader-bar"
-      [loading]="isLoading$ | async"
-    ></ix-fake-progress-bar>
-
-    <div class="dataset-tree-search">
-      <ix-search-input [disabled]="!dataSource" (search)="onSearch($event)"></ix-search-input>
-    </div>
+<div fxLayout="column" class="material mat-card" [class.is-empty]="dataSource.data.length === 0" *ngIf="dataSource.data.length === 0; else datasetContainer">
```

Comment:
So, if the block only appears when `dataSource.data.length === 0`, then the `is-empty` class is always going to be there.
Let's put `*ngIf` as first attribute.

Reply:
Fixed